airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/spec.json

```diff
@@ -16,35 +16,48 @@
     "additionalProperties": false,
     "properties": {
       "start_date": {
+        "title": "Start Date",
         "type": "string",
         "description": "UTC date in the format 2017-01-25. Any data before this date will not be replicated.",
-        "examples": ["2017-01-25T00:00:00Z"]
+        "examples": ["2017-01-25"]
       },
       "refresh_token": {
+        "title": "Refresh Token",
         "type": "string",
         "description": "The refresh token used obtained via authorization (can be passed to the client instead)"
       },
       "lwa_app_id": {
+        "title": "LWA APP ID",
         "type": "string",
         "description": "Your login with amazon app id"
       },
       "lwa_client_secret": {
+        "title": "LWA Client Secret",
         "type": "string",
         "description": "Your login with amazon client secret"
       },
       "aws_access_key": {
+        "title": "AWS Access Key",
         "type": "string",
         "description": "AWS USER ACCESS KEY"
       },
       "aws_secret_key": {
+        "title": "AWS Secret Key",
         "type": "string",
         "description": "AWS USER SECRET KEY"
       },
       "role_arn": {
+        "title": "Role ARN",
         "type": "string",
         "description": "The role’s arn (needs permission to “Assume Role” STS)"
       },
+      "seller_id": {
```

Comment:
what is this used for?

Reply:
spoke offline with Harshith: This is used to generate the refresh_token and is how amazon actually know what data to return in the query even though this id is not passed into the query at all. Because of this, it's actually useful to inject this data into the generated record so users know where the data came from.